src/backend/looped/self/persist_outfit.cpp

```diff
@@ -15,13 +15,13 @@ namespace big
 		int offset = 0;
 		switch (model)
 		{
-			case "mp_m_freemode_01"_J: break;
-			case "mp_f_freemode_01"_J:
-			{
-				offset = 1;
-				break;
-			}
-			default: return false; //For non-normal models
+		case "mp_m_freemode_01"_J: break;
+		case "mp_f_freemode_01"_J:
+		{
+			offset = 1;
+			break;
+		}
+		default: return false; //For non-normal models
```

Comment:
Unnecessary whitespace change.

Reply:
This is just how it should've been since the beginning, my IDE apply the clang format of the YimMenu project when I save the file.

Reply:
I think it looks stupid. The cases should be indented to the switches, since they belong to the switch's scope, not the other way around.

Reply:
I agree, but this is how the clang-format file was defined a long time ago now, any change to it would mean we'd need to do a PR that fix the files to the new formatting